test/e2e/list_test.go

```diff
@@ -0,0 +1,100 @@
+// Copyright 2020 VMware, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+package e2e
+
+import (
+	"strings"
+	"testing"
+	"time"
+
+	uitest "github.com/cppforlife/go-cli-ui/ui/test"
+	"github.com/stretchr/testify/require"
+)
+
+func TestAppListing(t *testing.T) {
```

Comment:
Should we call it `TestAppFilter` instead? (If so, we might wanna update the file name as well)

Reply:
Yeah make sense. made the changes.